streamalert_cli/_infrastructure/_include.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.3.0"
+      version = "~> 3.60.0"
```

Comment:
this terraform syntax should be avoided for the aws provider. the `~>` constraint only allows for the RIGHT MOST member of the tuple to increment (eg: in `3.N.n` only `n` can change) Effectively, this is actually `== 3.60.0` since this provider never publishes patch semvers.

we should update this to a constraint that more loosely allows incrementing, like: `>= 3.60.0`. possibly also restrict to `< 4.0.0`

also see the note here: https://www.terraform.io/docs/language/modules/develop/providers.html#provider-aliases-within-modules

>If you are writing a shared Terraform module, constrain only the minimum required provider version using a >= constraint.

Reply:
I have opted for the latter approach of constraining between two versions vs pinning to a specific version to allow the most flexibility.